src/layout/layout.js

```diff
@@ -303,7 +303,7 @@
 
           var drawerButtonIcon = document.createElement('i');
           drawerButtonIcon.classList.add(this.CssClasses_.ICON);
-          drawerButtonIcon.textContent = this.Constant_.MENU_ICON;
+          drawerButtonIcon.innerHTML = this.Constant_.MENU_ICON;
```

Comment:
I don't think a simple replace is good enough here. Way too much power with JS to sneak things in that still get computed. I'll look into finding a good alternative, otherwise we will need to revert and just leave the menu button as-is for 1.x for security purposes.